src/main/resources/application.yml

```diff
@@ -59,7 +59,7 @@ jobs:
 temporary-certificate:
   certificate-pdf:
     english:
-      path: "classpath:temporary-certificate-template/Temp Voter Authority Certificate (English).pdf"
+      path: "classpath:temporary-certificate-template/Temp Voter Authority Certificate (English) v1.pdf"
```

Comment:
Added versions to filenames so it's easy to know from which template a particular temporary certificate was generated.